src/components/Proposal/Predictions/PredictionButtons.tsx

```diff
@@ -258,7 +257,7 @@ class PredictionBox extends React.Component<IProps, IState> {
               }
             </span>
             : <span className={css.disabledPredictions}>
-               Predictions are disabled
+                { /* Predictions are disabled */ }
```

Comment:
?

Reply:
that text is superconfusing to me, every time I see it. I created an issue and will put it back for now